playbooks/openshift/templates/inventory.multimaster.j2

```diff
@@ -69,6 +69,7 @@ openshift_hosted_router_certificate={'certfile': '{{ certificate_crt }}', 'keyfi
 # deploy metrics service
 openshift_hosted_metrics_deploy={{ oso_deploy_metrics|default(false)|bool }}
 openshift_hosted_metrics_resolution=20s
+openshift_hosted_metrics_deployer_version={{ oso_metrics_tag|default(oso_image_tag|default('v'+oso_release|default('latest'))) }}
```

Comment:
There is quite a lot of nesting with defaults going on here. Do we need this level of granularity?